portlets/loginregister-portlet/src/main/java/org/xcolab/portlets/loginregister/singlesignon/OpenIdController.java

```diff
@@ -118,13 +118,15 @@ public void readOpenIdResponse(ActionRequest actionRequest, Model model, ActionR
             } catch (UserLocationNotResolvableException ignored) { }
 
             User user;
+            final HttpServletRequest httpReq = PortalUtil.getHttpServletRequest(actionRequest);
+            final String path = httpReq.getSession().getServletContext().getRealPath("/");
             try {
                 user = UserLocalServiceUtil.getUserByOpenId(
                         themeDisplay.getCompanyId(), openId);
                 portletSession.setAttribute(SSOKeys.OPEN_ID_LOGIN, user.getUserId(), PortletSession.APPLICATION_SCOPE);
 
                 actionResponse.sendRedirect(redirectUrl);
-                ImageUploadUtils.updateProfilePicture(user, profilePicURL);
+                ImageUploadUtils.updateProfilePicture(path, user, profilePicURL);
```

Comment:
This class broke by the introduction of https://github.com/CCI-MIT/XCoLab/commit/19e4ca04740a93f41a8af3c7ab849c524d085182